websites/Y/YouTube/presence.ts

```diff
@@ -227,6 +233,7 @@ presence.on("UpdateData", async () => {
 			);
 		}
 
+		if (!YoutubeShorts) uploaderShorts = void 0;
```

Comment:
Why not set it to null or undefined?

Reply:
eslint

Reply:
Don't think that that is the best way to fix it